httpd/default.toml

```diff
@@ -1 +1,47 @@
-listen = [ "80" ]
+#httpd.conf settings
+serveradmin = "you@example.com"
+servername = "localhost"
+serverport = "80"
+listen = ["80"]
+user = "hab"
+group = "hab"
+default_modules = [	"authn_file", "authn_core", "authz_host", "authz_groupfile", "authz_user", "authz_core",
+					"access_compat", "auth_basic", "reqtimeout", "filter", "mime", "log_config", "env", "headers",
+					"setenvif", "version", "ssl", "status", "autoindex", "dir", "alias"]
+
+#httpd-default settings
+timeout = 60
+keepalive = "On"
+keepaliverequests = 100
+keepalivetimeout = 5
+serversignature = "Off"
+servertokens = "Full"
+accessfilename = ".htaccess"
+canonicalname = "Off"
+hostnamelookups = "Off"
```

Comment:
I can see keeping the "default" settings at the top level. However, for clarity it might be good to use namespaces for the others, below.